.github/workflows/build-container.yml

```diff
@@ -39,12 +39,18 @@ jobs:
         tag_list=$docker_repo:$REPO,$docker_repo:$REPO-$TIMESTAMP
         echo "::set-output name=taglist::$tag_list"
 
-    - name: Build and push Docker images
-      uses: docker/build-push-action@v2.2.0
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
-        repository: opensciencegrid/osgvo-docker-pilot
```

Comment:
Ok i thought there was some reason we needed to specify `repository` explicitly in `build-push-action@v2.2.0`

Reply:
Nope, the repo has to be specified as part of each tag now